backport

```diff
@@ -7,11 +7,12 @@ function usage() {
   echo "Then it pushes the newly created branch and creates a new PR."
   echo
   echo "Usage: "
-  echo "$(basename "$0") <source-ref> <target-branch> [-c | --continue] [-l | --local]"
+  echo "$(basename "$0") <source-ref> <target-branch> [-c | --continue] [-l | --local] [-ni | --non-interactive]"
   echo
   echo "Options:"
-  echo "  -l, --local    Skip pushing the branch and creating the PR"
-  echo "  -c, --continue Continue backporting after fixing cherry-pick conflict"
+  echo "  -l,  --local           Skip pushing the branch and creating the PR"
+  echo "  -c,  --continue        Continue backporting after fixing cherry-pick conflict"
+  echo "  -ni, --non-interactive Continue backporting after fixing cherry-pick conflict"
```

Comment:
The description of the flag is wrong

Reply:
> The description of the flag is wrong

🤦

68469e53c0058be5df90600286b3273c4a21d97f